### PR TITLE
Improve documentation ahead of 2.1.6 release

### PR DESCRIPTION
**Pull request by kgaillot**

### CODE REVIEW

**Comment by kgaillot** [comment]
@clumens, here's another one when you get a chance. No rush

File: doc/sphinx/Pacemaker_Administration/troubleshooting.rst [hunk 1]
```diff
@@ -58,16 +88,36 @@ state at that moment (the CIB). This file can help determine why particular
...
+explanation, try to answer questions like the following based on log messages:
+
+* When and what was the last successful message on the node itself, or about
+  that node in the other nodes' logs?
+* Did pacemaker-controld on the other nodes notice the node leave?
+* Did pacemaker-controld on the DC invoke the scheduler and schedule a new
+  transition?
+* Did the transition include fencing the failed node?
+* Was the fencing attempted?
+* Did the fencing succeed?
```
Comment (clumens) [inline doc/sphinx/Pacemaker_Administration/troubleshooting.rst:108]: I think these two read more naturally as "Was fencing attempted?" and "Did fencing succeed?"

File: daemons/fenced/README.md
Comment (clumens) [inline daemons/fenced/README.md:1]: You'll want to remove it from EXTRA_DIST in Makefile.am, too.

File: etc/sysconfig/pacemaker [hunk 2]
```diff
@@ -1,173 +1,337 @@
...
+#
+# Default: PCMK_trace_formats=""
+# Example: PCMK_trace_formats="TLS handshake failed: %s (%d)"
+
+# PCMK_trace_tags (Advanced Use Only)
+#
+# Log trace severity messages related to these (comma-separated) resource IDs.
+#
+# Default: PCMK_trace_tags=""
+# Example: PCMK_trace_tags="client-ip,dbfs"
```
[the remark this answers is not in the document]
Comment (kgaillot) [reply etc/sysconfig/pacemaker]: Good point

File: doc/sphinx/Pacemaker_Administration/troubleshooting.rst (hunk 1 above)
Comment (kgaillot) [reply doc/sphinx/Pacemaker_Administration/troubleshooting.rst:108]: thanks

File: daemons/fenced/README.md
Comment (kgaillot) [reply daemons/fenced/README.md:1]: whoops

**Comment by kgaillot** [comment]
Added commits on top to address review

**Pull request merged**